skiko/src/commonMain/kotlin/org/jetbrains/skia/Canvas.kt

```diff
@@ -1501,7 +1501,9 @@ private external fun _nDrawPicture(ptr: NativePointer, picturePtr: NativePointer
 
 
 @ExternalSymbolName("org_jetbrains_skia_Canvas__1nDrawVertices")
-private external fun _nDrawVertices(
+// Not private as it is explicitly called in
+// compose/ui/ui-graphics/src/*/kotlin/androidx/compose/ui/graphics/*Canvas.*.kt
```

Comment:
We’d better expose it as an API.

Reply:
created Canvas::drawVertices

Reply:
Nah, just dropped it from the PR for now. Will do a separate PR on _nDrawVertices